test/README.md

```diff
@@ -206,6 +217,65 @@ from the local directory without installing it:
 * `make build` - will build everything under `cmd/`
 * `./osbuild-composer-cli-tests` - will execute the freshly built integration test suite
 
+### Cloud API integration testing
```

Comment:
Love this documentation! Another way to test this would be to use `docker-compose up --build` (introduced by @achilleas-k) rather than building/installing on your local machine. To make that really smooth, we may want to introduce a third container in which to actually run the test scripts, but not sure if that's necessary.
No need to change this or look into the docker stuff now though, just wanted to mention it in case you didn't know about it.

Reply:
Thanks for the note. I know about the containers work, but didn't use it much because it didn't work with `podman-compose` last time I tried it. And unfortunately I don't have docker installed, just podman. Nevertheless I definitely see how running the whole setup in separate containers could be beneficial for testing and debugging.

For now I mostly copied the wording from the composer-cli testing section which talked about running the test case locally. Personally I found running the cloudapi integration test directly on the GCP instance very useful, but it would probably run just OK in containers on the instance.

I may get back to the containers work and try to make it work with `podman-compose`.